srcpkgs/factor/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'factor'
+pkgname=factor
+version=0.97
+revision=1
+only_for_archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="unzip pkg-config"
+makedepends="gtkglext-devel"
+nostrip_files="a.elf"
+short_desc="Concatenative programming language, similar to Forth"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD-2-Clause"
+homepage="http://factorcode.org/"
+distfiles="http://downloads.factorcode.org/releases/${version}/factor-src-${version}.zip"
+checksum=fd75e765590691c89b866e5265ae16bfb36d0e28eed095702ae1e206114663b8
+
+post_extract() {
+	mv "${XBPS_BUILDDIR}/factor" "${wrksrc}"
+}
+
+post_build() {
+	image=''
+
+	case "${XBPS_MACHINE}" in
```

Comment:
shouldn't this be `XBPS_TARGET_MACHINE`

Reply:
Ugh. Sloppy mistake. Thanks for catching it!